android/apollo/src/com/andrew/apollo/loaders/RecentLoader.java

```diff
@@ -49,30 +50,14 @@ private static Cursor makeRecentCursor(final Context context) {
         return RecentStore
                 .getInstance(context)
                 .getReadableDatabase()
-                .query(RecentStoreColumns.NAME,
+                .query(RecentStore.RecentStoreColumns.TABLE_NAME,
```

Comment:
`TABLE_NAME` should not be defined inside `RecentStoreColumns` since it's not a column.
Why not just `RecentStore.TABLE_NAME`? KISS.